build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -789,6 +789,51 @@ export const handlers: Handler[] = [
 			}
 		},
 	},
+	{
+		name: "npm-package-json-test-scripts-split",
+		match,
+		handler: (file, root) => {
+			let json;
+
+			try {
+				json = JSON.parse(readFile(file));
+			} catch (err) {
+				return "Error parsing JSON file: " + file;
+			}
+
+			const scripts = json.scripts;
+			if (scripts === undefined) {
+				return undefined;
+			}
+			const testScript = scripts["test"];
+
+			const splitTestScriptNames = ["test:mocha", "test:jest", "test:realsvc"];
+
+			if (testScript === undefined) {
+				if (splitTestScriptNames.some((name) => scripts[name] !== undefined)) {
+					return "Missing 'test' scripts";
+				}
+				return undefined;
+			}
+
+			const actualSplitTestScriptNames = splitTestScriptNames.filter(
+				(name) => scripts[name] !== undefined,
+			);
+
+			if (actualSplitTestScriptNames.length === 0) {
+				if (!testScript.startsWith("echo ")) {
+					return "Missing split test scripts";
```

Comment:
If I understood things right, this is the action someone would need to take if the policy complained?

```suggestion
					return "Missing split test scripts. The 'test' script must call one or more \"split\" scripts like 'test:mocha', 'test:jest', or 'test:realsvc'.";
```